src/core/pages/PatientSummary/index.js

```diff
@@ -11,18 +11,15 @@ import {
     synopsisContactsAction,
     synopsisMedicationsAction,
     synopsisProblemsAction } from "../../actions/synopsisActions";
-import { synopsisData, getSynopsisProps, SHOW_ALL } from "./config";
+import { synopsisData, getSynopsisProps } from "./config";
 import SettingsDialog from "./SettingsDialog";
 import Breadcrumbs from "../../common/Breadcrumbs";
-
 import { themeCommonElements } from "../../../version/config/theme.config";
 import { nonCoreSynopsisActions } from "../../../version/config/nonCoreSynopsis";
+import { getSummaryContainerStyles } from "./functions";
 
 const styles = theme => ({
-    summaryContainer: {
-        margin: 0,
-        width: "100%",
-    },
+    summaryContainer: getSummaryContainerStyles(synopsisData),
```

Comment:
if `getSummaryContainerStyles` is always called with `synopsisData`, why don't you import `synopsisData` directly in `getSummaryContainerStyles`?